### PR TITLE
Add Steps for Azure NFS Installation

### LINKED ISSUE #696: NFS Installation on Azure clarification
**Affected Document:**
https://docs.astronomer.io/software/deploy-nfs

**Problem Type (Priority):**

Azure NFS volumes should have more guidance then referring just to the Azure documentation

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**
For this to work with Azure you need to use premium Azure file share, TLS requirement disabled, and enable NFS. There is an internal guide that should be incorporated into this document to help customers get this feature working quickly.

**Additional Notes:**

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #696

### CODE REVIEW

**Review by cmarteepants — CHANGES_REQUESTED** [review]
We should either 
1. Remove steps 7 to 10, or;
2. Put those steps in a different section saying that as an example for getting dags into the NFS share. And then add the remaining steps of setting up a service account, and using that service account to copy the dags into the pod

File: software/deploy-nfs.md [hunk 1]
```diff
@@ -58,3 +59,87 @@ Workspace editors can configure a new or existing Airflow Deployment to use a pr
 5. Save your changes.
 
 > **Note:** You can also use the  Astro CLI to configure NFS volumes. To do so, specify the `--nfs-location` flag when running [`astro deployment create`](cli-reference.md#astro-deployment-create) or [`astro deployment update`](cli-reference.md#astro-deployment-update).
+
+## Create and mount an Azure file share
+
+Create and mount an Azure file share to deploy DAGs to an Airflow Deployment on Astronomer. For additional information about this process, see [File Storage](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-how-to-create-nfs-shares?tabs=azure-portal).
```
Comment (rbankston) [inline software/deploy-nfs.md:65]: would like to see a change of [File Storage] to [Premium File Storage]

Comment (cmarteepants) [reply software/deploy-nfs.md:65]: Is that because NFS is only available with Premium File Storage?

Comment (kwcanuck) [reply software/deploy-nfs.md]: @cmarteepants and @rbankston I have updated the procedure based on your feedback. If there are specific changes or additions you want:

1. Click **Files changed**.

2. Scroll to the location where you want to add or edit text and click the **Plus** (+) icon.

image

3. Click **Add a suggestion**.

image

4. Add or edit text and then click **Add single comment**.

If there is specific text  or procedure steps you want added, please provide the information, or direct me to a location where it is provided. 

Please review the changes and provide your edits, or approval.

Thanks!

**Review by cmarteepants — APPROVED** [review]
This is great, thank you!

**Comment by kwcanuck** [comment]
@jwitz ready for your review

**Review by rbankston — APPROVED** [review]
Thank you Rob. That looks good to me

**Comment by kwcanuck** [comment]
@jwitz this is ready for review.

File: software/deploy-nfs.md [hunk 2]
```diff
@@ -58,3 +59,28 @@ Workspace editors can configure a new or existing Airflow Deployment to use a pr
 5. Save your changes.
 
 > **Note:** You can also use the  Astro CLI to configure NFS volumes. To do so, specify the `--nfs-location` flag when running [`astro deployment create`](cli-reference.md#astro-deployment-create) or [`astro deployment update`](cli-reference.md#astro-deployment-update).
+
+## Create and mount an Azure file share
+
+Create and mount an Azure file share to deploy DAGs to an Airflow Deployment on Astronomer. For additional information about this process, see [Premium File Storage](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-how-to-create-nfs-shares?tabs=azure-portal).
```
Comment (jwitz) [inline software/deploy-nfs.md:65]:
```suggestion
Create and mount an Azure file share to deploy DAGs from an NFS volume to an Astronomer installation on Azure. For additional information about this process, see [Premium File Storage](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-how-to-create-nfs-shares?tabs=azure-portal).
```

File: software/deploy-nfs.md [hunk 3]
```diff
@@ -58,3 +59,28 @@ Workspace editors can configure a new or existing Airflow Deployment to use a pr
...
+1. Update your `config.yaml` file with the following values:
+
+  ```yaml
+        houston:
+          config:
+              deployments:
+                configureDagDeployment: true
+                nfsMountDagDeployment: true
+  ```
+2. Save the `config.yaml` file and push the configuration change to your platform. See [Apply a config change](apply-platform-config.md). THis process only needs to be completed once for each Software installation.
```
Comment (jwitz) [inline software/deploy-nfs.md:76]:
```suggestion

2. Save the `config.yaml` file and push the configuration change to your platform. See [Apply a config change](apply-platform-config.md). This process only needs to be completed once for each Software installation.
```

**Review by jwitz — APPROVED** [review]

**Pull request merged**